### PR TITLE
feat: venues and sponsors

### PR DESCRIPTION
**Pull request by kateinkim**
# Description

[Trello Card](70-real-data-sponsors)

Summary of the changes: 

- Added and updated the `Sponsor` and `Venue` types
- Updated the venue screen to have real data
- Changed the gold sponsor logo size to be smaller than platinum's per design
- Changed the prop names of the `TierLevelProps` interface to match the CMS

# Graphics

https://user-images.githubusercontent.com/53795920/213507526-d1bc6935-caea-4ea5-a178-93737cfecdac.mp4

# Checklist:

- [ ] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [ ] I have run tests and linter

### CODE REVIEW

File: app/screens/VenueScreen/VenueScreen.tsx [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 
-const irImage1 = require("../../../assets/images/info-ir1.png")
-const irImage2 = require("../../../assets/images/info-ir2.png")
+const sponsorTiers = Object.values(WEBFLOW_MAP.sponsorTier)
+type Tiers = typeof sponsorTiers[number]
+
+const initialTiers = sponsorTiers.reduce<Record<Tiers, RawSponsor[]>>(
+  (acc, tier) => ({ ...acc, [tier]: [] }),
+  {} as Record<Tiers, RawSponsor[]>,
+)
```
Comment (kateinkim) [inline app/screens/VenueScreen/VenueScreen.tsx:20]: This was a blocker for some time that I had to ask my 6'5" elf to help me with 😅

**Comment by kateinkim** [comment]
[Real Data: Sponsors](https://trello.com/c/pQpjo1Xv/70-real-data-sponsors)

File: app/screens/VenueScreen/VenueScreen.tsx (hunk 1 above)
Comment (robinheinze) [reply app/screens/VenueScreen/VenueScreen.tsx:20]: LOL

File: app/screens/VenueScreen/VenueScreen.tsx [hunk 2]
```diff
@@ -26,103 +35,75 @@ const SponsorSection = ({ item }) => {
...
+  return [
+    {
+      title: translate("venueScreen.conferenceAndWorkshopVenues"),
+      renderItem: Workshops,
+      data: [
+        venues?.map((venue) => ({
+          image: { uri: venue["venue-image-s"][0]?.url },
+          subtitle: venue.name,
+          meta: `${WEBFLOW_MAP.venueTag[venue.tag]} • ${
+            WEBFLOW_MAP.venueTag[venue.tag] === "Workshop" ? "May 17" : "May 18-19"
```
[the remark this answers is not in the document]
Comment (robinheinze) [reply app/screens/VenueScreen/VenueScreen.tsx:59]: I think this is fine

**Review by robinheinze — APPROVED** [review]
Looks good!

**Pull request merged**